integration_tests/main.rs

```diff
@@ -288,10 +474,34 @@ impl Operation {
 
 #[derive(Debug)]
 struct Outcome {
-    customer_status: CustomerStatus,
-    merchant_status: MerchantStatus,
     /// Which process, if any, had an error? Assumes that exactly one party will error.
     error: Option<Party>,
+    channel_outcome: Option<ChannelOutcome>,
+}
+
+#[derive(Debug)]
+struct ChannelOutcome {
+    customer_status: CustomerStatus,
+    merchant_status: MerchantStatus,
+    customer_balance: CustomerBalance,
+    merchant_balance: MerchantBalance,
+}
+
+/// Helper function to convert human XTZ amount into a `CustomerBalance`.
+fn into_customer_balance(amount: u64) -> CustomerBalance {
```

Comment:
todo: This is my fault, but can you please rename this and the merchant balance version to be `to_customer_balance`?

This is following the Rust naming conventions: `u64` and `CustomerBalance` are both owned (not references) and `Copy`. https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv